From 0f4d508ca3dc0eac4ef4ac85190da58285f1580f Mon Sep 17 00:00:00 2001
From: Stefan Wahren <wahrenst@gmx.net>
Date: Fri, 27 Dec 2019 12:01:17 +0100
Subject: [PATCH] ARM: dts: bcm2838: Add upstream RNG compatible

This adds the ability to use the RNG with an upstream kernel.
Keep the old one for backward compatibility.

Signed-off-by: Stefan Wahren <wahrenst@gmx.net>
---
 arch/arm/boot/dts/bcm2838.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/bcm2838.dtsi
+++ b/arch/arm/boot/dts/bcm2838.dtsi
@@ -682,7 +682,7 @@
 };
 
 &rng {
-	compatible = "brcm,bcm2838-rng200";
+	compatible = "brcm,bcm2711-rng200", "brcm,bcm2838-rng200";
 };
 
 &sdhost {
